G-Research.OpenTelemetry.Processor.Partial.Tests/PartialActivityProcessorTests.cs

```diff
@@ -1,10 +1,9 @@
 ﻿using System.Diagnostics;
 using OpenTelemetry.Exporter;
-using OpenTelemetry.Exporter.Partial;
 using OpenTelemetry.Logs;
 using Xunit;
 
-namespace OpenTelemetry.Export.Partial.Tests
+namespace Processor.Partial.Tests
```

Comment:
I would suggest using `GR.OpenTelemetry.Processor.Partial` as the root namespace, like it's done in other 3rd-party OTEL libs

Reply:
i renamed it like that, not sure why it's not seen in the pr, i see it locally.